enterprise/migrations/0006_auto_20161020_1023.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
@mulby this is the plan. I've kept them separate for two reasons:

1. It is easier for reviewers to keep track of what's changed after their last review.
2. It is a bit simpler in terms of development process (keeping them squashed to single migration would require migrating back to "zero" migration and then forward again each time something has changed; as opposed to forward-only migration in this case).